server/lib/configuration.js

```diff
@@ -274,6 +286,10 @@ var conf = module.exports = convict({
 // on the path, we'll use that, otherwise we'll name it 'ephemeral'.
 conf.set('process_type', path.basename(process.argv[1], '.js'));
 
+if (conf.get('env') == 'development') {
```

Comment:
Re: `===` not being caught by JSHint,

It seems like we're not linting the "server/**" directory w/ JSHint. Adding this to the grunttasks/jshint.js gives me about 70 errors/warnings:
```js
    server: {
      src: [
        'server/**/*.js'
      ]
    },
```